lib/service/service.go

```diff
@@ -254,6 +254,9 @@ const (
 
 	// TeleportOKEvent is emitted whenever a service is operating normally.
 	TeleportOKEvent = "TeleportOKEvent"
+
+	// externalUpgraderEnv is the external upgrader environment variable.
```

Comment:
This comment doesn't really say anything that the variable name doesn't already say. Can we make it more helpful to readers who don't know what an external upgrader is?